README.md

```diff
@@ -27,7 +27,7 @@ Development environment can help you debug problems in your application in the s
 The Perl container is set up so that Apache will load .conf files located within the <code>cfg</code> directory of the application's root.  This is useful if you are configuring your application with a database backend and would want to pass through your environment variables to mod_perl with <code>PerlPassEnv</code>.
 
 ###Installation: 
-These steps assume your OpenShift deployment has the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html)
+These steps assume your OpenShift deployment has the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html).    If you are defining the set of ImageStreams now, remember to pass in the proper credentials via the --config option.
```

Comment:
proper cluster-admin credentials and to create the ImageStreams in the 'openshift' namespace